DIRECTORY.md

```diff
@@ -0,0 +1,159 @@
+# List of all files
+
+## DataStructures
+  * AATree
+    * [Aatree](https://github.com/TheAlgorithms/C-Sharp/blob/master/DataStructures/AATree/AATree.cs)
```

Comment:
```suggestion
    * [AA Tree](https://github.com/TheAlgorithms/C-Sharp/blob/master/DataStructures/AATree/AATree.cs)
```
Please check the casing and spacing of generated list items.

Reply:
Fixed the casing, the spaces are more difficult because they aren't in the file name